packages/create-gatsby/src/index.ts

```diff
@@ -0,0 +1,59 @@
+import { prompt } from "enquirer"
+import questions from "./questions.json"
+import { initStarter } from "./init-starter"
+
+interface IAnswers {
+  project: string
+  styling?: string
+  cms?: string
+  features?: Array<string>
+}
+
+export async function run(): Promise<void> {
+  console.log(
+    `Welcome to Gatsby! This command will generate a new Gatsby site for you with the setup you select. Let's answer some questions:`
+  )
+  const data = await prompt<IAnswers>(questions)
+
+  const messages: Array<string> = [
+    `🛠  Create a new Gatsby site called ${data.project}`,
+  ]
+  if (data.cms) {
```

Comment:
Interestingly, enquirer returns the `message` instead of the `value`, so this will actually always be true.

```
if (data.cms) // "No (or I'll add it later)" is the message which is truthy
```

It would appear [others are confused by this](https://github.com/enquirer/enquirer/issues/121) like I was.

Looks like there is a result function you can add to the question but that means we'll probably have to move the questions into a TypeScript file too? 🤔

Reply:
The naming of the properties is pretty confusing, and I keep getting it wrong!